<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerDlg</class>
 <widget class="QDialog" name="registerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>42</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>43</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="endDateLineEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:6px;
font:11pt;
font-family:Microsoft Yahei;
border:1px solid #bfd2e3;
}
QLineEdit:hover{
border-width:2px;
border-color:lightblue;
}
QLineEdit:focus {
border-width:2px;
border-color:skyblue;
}</string>
         </property>
         <property name="placeholderText">
          <string>格式：2018-08-08</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="stopLabel">
         <property name="styleSheet">
          <string notr="true">color: #31576d;
border-radius:6px;
font:12pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>结束时间：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="operationLabel">
         <property name="styleSheet">
          <string notr="true">color: #31576d;
border-radius:6px;
font:12pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>立即生效：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QRadioButton" name="enableRadioButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font:bold;
font:11pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>是</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="startDateLineEdit">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:6px;
font:11pt;
font-family:Microsoft Yahei;
border:1px solid #bfd2e3;
}
QLineEdit:hover{
border-width:2px;
border-color:lightblue;
}
QLineEdit:focus {
border-width:2px;
border-color:skyblue;
}</string>
         </property>
         <property name="placeholderText">
          <string>格式：2018-01-01</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QRadioButton" name="disenableRadioButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font:bold;
font:11pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>否</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: #31576d;
border-radius:6px;
font:12pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:6px;
font:11pt;
font-family:Microsoft Yahei;
border:1px solid #bfd2e3;
}
QLineEdit:hover{
border-width:2px;
border-color:lightblue;
}
QLineEdit:focus {
border-width:2px;
border-color:skyblue;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
         <property name="placeholderText">
          <string>设置密码</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: #31576d;
border-radius:6px;
font:12pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>密码：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="userLineEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:6px;
font:11pt;
font-family:Microsoft Yahei;
border:1px solid #bfd2e3;
}
QLineEdit:hover{
border-width:2px;
border-color:lightblue;
}
QLineEdit:focus {
border-width:2px;
border-color:skyblue;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入用户名</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="mailLineEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:6px;
font:11pt;
font-family:Microsoft Yahei;
border:1px solid #bfd2e3;
}
QLineEdit:hover{
border-width:2px;
border-color:lightblue;
}
QLineEdit:focus {
border-width:2px;
border-color:skyblue;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入邮箱地址</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color: #31576d;
border-radius:6px;
font:12pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>邮箱：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: #31576d;
border-radius:6px;
font:12pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>电话：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="phoneLineEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:6px;
font:11pt;
font-family:Microsoft Yahei;
border:1px solid #bfd2e3;
}
QLineEdit:hover{
border-width:2px;
border-color:lightblue;
}
QLineEdit:focus {
border-width:2px;
border-color:skyblue;
}</string>
         </property>
         <property name="placeholderText">
          <string>输入11位电话号码</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="startLabel">
         <property name="styleSheet">
          <string notr="true">color: #31576d;
border-radius:6px;
font:12pt;
font-family:微软雅黑;</string>
         </property>
         <property name="text">
          <string>开始时间：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Yahei</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font:15pt;
font-family:Microsoft Yahei;</string>
       </property>
       <property name="text">
        <string>填写账户信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="registerPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>38</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(38,188,213);
    border-radius:5px;
	font:bold;
	font: 13pt;
}

QPushButton:hover
{
    background-color:deepskyblue;
}

QPushButton:pressed
{
    background-color:deepskyblue;
    padding-left:4px;
    padding-top:4px;
}
QPushButton:unpressed
{
    background-color:rgb(0 , 100 , 0);
    padding-left:4px;
    padding-top:4px;
}</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>mailLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>startDateLineEdit</tabstop>
  <tabstop>endDateLineEdit</tabstop>
  <tabstop>enableRadioButton</tabstop>
  <tabstop>disenableRadioButton</tabstop>
  <tabstop>registerPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
